source/MaterialXGenGlsl/Nodes/SurfaceNodeMaya.h

```diff
@@ -0,0 +1,30 @@
+//
+// TM & (c) 2017 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#ifndef MATERIALX_SURFACENODEMAYA_H
+#define MATERIALX_SURFACENODEMAYA_H
+
+#include <MaterialXGenGlsl/Nodes/SurfaceNodeGlsl.h>
+#include <MaterialXGenGlsl/GlslShaderGenerator.h>
+
+namespace MaterialX
+{
+
+/// Surface node implementation for GLSL
+class SurfaceNodeMaya : public SurfaceNodeGlsl
```

Comment:
Since this is a Maya specific class specialization only used in MaterialXGenOgsXml it would be better to place it in that library. You could move it to `MaterialXGenOgsXml/Nodes/SurfaceNodeMaya.{h,cpp}`

Reply:
Ah yes. I agree with Niklas that this should not be here.